main.py

```diff
@@ -32,9 +32,6 @@ def pagecountbynamespace( self ):
 	def pages_with_most_revisions( self ):
 		self.rep.pages_with_most_revisions()
 
-	def blank_pages( self ):
```

Comment:
You have written "fixed in BernsteinBot" in remove commit message, but is it possible to keep the function, if it's functional, in this repo ?